README.md

```diff
@@ -18,14 +18,13 @@ $ npm install
 
 ### Compiles and hot-reloads for development
 ```bash
-# serve with hot reload at localhost:3000
+# serve with hot reload at localhost:5173
```

Comment:
is there a reason this changed? On Provident I think this happened to me because I was using NPM when i should have been using Yarn.

Reply:
it's the default for vite. it use to be 3000
https://vitejs.dev/config/server-options.html